src/components/navigation-header/components/home-page-content/home-page-content.module.css

```diff
@@ -1,3 +1,13 @@
 .siteLogo {
   display: flex;
+
+  /* Note: padding here necessary to match height of .productLogo
+     in navigation-header.module.css */
+  padding-top: 4px;
+  padding-bottom: 4px;
+
+  & svg {
+    height: 32px;
```

Comment:
This is intended to fix height inconsistency issues on the home page, where the logo takes on a different height (per design specs).